CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,3 @@
+# Community Code of Conduct
+
+Cilium follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
at the risk of the link become 404, shouldn't we copy paste the entire text into cilium?

Reply:
All good points, I've included it and adjusted contact details.